### PR TITLE
prepare release 2.0.2

### PR DESCRIPTION
**Pull request by chaudum**

### CODE REVIEW

File: blackbox/docs/release_notes/2.0.2.txt [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+ - Fixed an issue that resulted in rows being unable to be queried by primary
+   keys, when the order of the primary key columns on insert differed from
+   the order of the primary key columns on the table definition.
+   Note: If records are already inserted by using a different primary key
+   column order, they must be re-inserted, otherwise queries will still fail
+   for these rows.
+
+ - Fixed an issue that could cause ``DELETE`` by query  and ``UPDATE``
+   statements to fail on datasets larger than 10_000 rows.
```
Comment (Autophagy) [inline blackbox/docs/release_notes/2.0.2.txt]: ~~10_000~~ 10,000 (unless this is how we usually do it?)

File: blackbox/docs/release_notes/2.0.2.txt [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+   led to killed nodes in the cluster, the issue is detected and the query is
+   killed without affecting the cluster.
+
+ - Fixed an issue that could cause ``DELETE`` statements to fail instead of
+   "not matching" if there was a ``_version`` column in the ``WHERE`` clause.
+
+ - Fixed an error handling issue that could lead to the termination of a node
+   in very rare cases. (Usually if a user invoked the ``KILL`` statement)
+
+ - Fixed issue where bulk operations like ``insert from dynamic queries`` and
```
Comment (Autophagy) [inline blackbox/docs/release_notes/2.0.2.txt]: ~~``insert from dynamic queries``~~ ``INSERT`` from dynamic queries

File: blackbox/docs/release_notes/2.0.2.txt [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+   ``COPY FROM`` did not stop after being killed.
+
+ - ``CREATE USER`` and ``DROP USER`` statements will now only respond after all
+   nodes in the cluster processed the change.
```
Comment (Autophagy) [inline blackbox/docs/release_notes/2.0.2.txt]: ~~processed~~ have processed

File: blackbox/docs/release_notes/2.0.2.txt (hunk 1 above)
Comment (chaudum) [inline blackbox/docs/release_notes/2.0.2.txt]: 10_000 is programmer style, 10,000 is more correct

**Pull request merged**